cmd/user/internal/interfaces/http/router.go

```diff
@@ -51,10 +51,13 @@ func NewRouter(logger *log.Logger, repository user_persistence.UserRepository, c
 	router.POST("/v1/google/callback", handlers.BuildSocialAuthHandler(googleAPIURL, commandBus, user.RegisterUserWithGoogle, secretKey, oauth2Config))
 	router.POST("/v1/facebook/callback", handlers.BuildSocialAuthHandler(facebookAPIURL, commandBus, user.RegisterUserWithFacebook, secretKey, oauth2Config))
 
-	// User routes
-	router.POST("/v1/dispatch/{command}", handlers.BuildCommandDispatchHandler(commandBus))
+	// Protected User routes
```

Comment:
Merge master and resolve conflicts (I merged https://github.com/vardius/go-api-boilerplate/pull/43)